src/Covid19Dashboard/ChartCarousel/ChartCarousel.less

```diff
@@ -44,24 +44,52 @@
 
 /** override some style for carousel's arrow and dots **/
 
+/* dots */
+.ant-carousel .slick-dots li button {
```

Comment:
I'd prefer to add a parent class so these changes are only applied to the carousel in covid19dashboard

Reply:
i think these should be global so it works the same everywhere (ill look for more environments using it and keep testing so i make sure it works everywhere)

Reply:
i implemented the css this way so that the buttons would be more accessible with bigger git states while still looking the same